.github/workflows/main.yaml

```diff
@@ -18,29 +18,34 @@ jobs:
         with:
           node-version: 22
       # Set up the build environment
-      - run: npm ci --prefix "./app/"
+      - run: npm ci
 
       # Build
-      - name: Copy our kickstart file over so it gets picked up
-        run: mkdir faDockerComposeFilePath/ && cp -r kickstart faDockerComposeFilePath/ && cp kickstart/kickstart.json faDockerComposeFilePath/kickstart/k2.json
+      - name: Checkout out the example kickstarts
+        uses: actions/checkout@v4
```

Comment:
We are pinning out github actions for security purposes. See https://github.com/FusionAuth/fusionauth-site/blob/main/.github/workflows/apicheck.yml#L21 for example.